pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/bindings/LTensorTNCudaBindings.hpp

```diff
@@ -47,12 +48,66 @@ namespace Pennylane::LightningTensor::TNCuda {
 using TensorNetBackends =
     Pennylane::Util::TypeList<MPSTNCuda<float>, MPSTNCuda<double>, void>;
 
+/**
+ * @brief Register controlled matrix kernel.
+ */
+template <class TensorNet>
```

Comment:
Do we need to use `T` for the sake of consistency?
```suggestion
template <class TensorNetT>
```

Reply:
Feel free to address this in the next PR.